lib/Dancer2/CLI/Gen.pm

```diff
@@ -200,15 +197,15 @@ sub _copy_templates {
             next unless ( $res eq 'y' ) or ( $res eq 'a' );
         }
 
-        my $to_dir = dirname( $to );
-        if ( ! -d $to_dir ) {
+        my $to_dir = path( $to )->parent;
+        if ( ! $to_dir->is_dir ) {
             print "+ $to_dir\n";
-            mkpath $to_dir or die "could not mkpath $to_dir: $!";
+            $to_dir->mkpath or die "could not mkpath $to_dir: $!";
         }
 
-        my $to_file = basename($to);
-        my $ex      = ($to_file =~ s/^\+//);
-        $to         = catfile($to_dir, $to_file) if $ex;
+        my $to_file = path( $to )->basename;
+        my $ex      = ( $to_file =~ s/^\+// );
+        $to         = path( $to_dir)->child( $to_file ) if $ex; # BUGGED
```

Comment:
list again, and why is this "BUGGED"?

Reply:
It was before I pushed it. Then promptly forgot to drop my comment. Fixed. Oops :)